karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(config) {
 
     webpack: webpackTestConfig,
     webpackServer: {
-      noInfo: true, // please don't spam the console when running in karma!
+      noInfo: false, // please don't spam the console when running in karma!
```

Comment:
? :)

Reply:
@bergquist Not sure if we really want that tbh, I can turn it off again. It's a change from one of Torkels commits.